packages/schemas/package.json

```diff
@@ -22,5 +28,17 @@
 		"typescript": "catalog:",
 		"vitest": "^1.3.1"
 	},
+	"preconstruct": {
+		"entrypoints": [
+			"index.ts",
+			"schemas.ts"
+		],
```

Comment:
easy to miss, but I added an extra entrypoint `schemas.ts` and thus export `schemas/schemas`, because in some places we were importing things from `schema/src/schemas`, which is not what we are supposed to do.